client/src/app/shared/components/markdown-text/markdown-parser.ts

```diff
@@ -3,21 +3,32 @@ import hljs from 'highlight.js/lib/core';
 import python from 'highlight.js/lib/languages/python';
 import plaintext from 'highlight.js/lib/languages/plaintext';
 
-import { IMarkdownParserOptions } from './index';
-
 hljs.registerLanguage('python', python);
 hljs.registerLanguage('plaintext', plaintext);
 
+export interface IMarkdownParserOptions {
+  highlight: boolean;
+}
+
+export interface IMarkdownParser {
+  parse: (markdownTest: string, options?: IMarkdownParserOptions) => Promise<string>;
+}
+
 const DEFAULT_OPTIONS: IMarkdownParserOptions = {
   highlight: true,
 };
 
 export async function parse(text: string, options: IMarkdownParserOptions = DEFAULT_OPTIONS): Promise<string> {
   const noYamlHeaderText = cutYamlHeader(text);
-  return marked.parse(noYamlHeaderText, options.highlight ? highlightConfig : null);
+  return marked.parse(noYamlHeaderText, options.highlight ? highlightConfig : commonConfig);
 }
 
+const commonConfig = {
+  breaks: true,
+};
+
 const highlightConfig = {
+  ...commonConfig,
```

Comment:
I would be better do not link this configs

Reply:
Fixed